lib/govuk_index/presenters/details_presenter.rb

```diff
@@ -19,11 +19,13 @@ def contact_groups
     def latest_change_note
       return nil if details["change_notes"].nil? || details["change_notes"].empty?
 
-      note_info = details["change_notes"]
-        .sort_by { |note| DateTime.parse(note["published_at"]) }
-        .last
+      unless format == "manual"
```

Comment:
Would it be safer to lock this down to the only format that uses change notes? I think it's just HMRC manuals.